src/ml_gym/util/util.py

```diff
@@ -191,15 +202,26 @@ def update_dataset_details(exp_config: dict) -> DatasetDetails:
                     obj (DatasetDetails): initialized dataset details object.
             """
             try:
-                dataset_splits = {"split_config": exp_config["dataset_iterators"]["config"]["split_configs"], 
-                              "splits_percentage": exp_config["splitted_dataset_iterators"]["config"]["split_configs"]}
+                splits_percentage = []
+                split_subscription = set()
+                find_key = "dataset_iterator"
+                for key in exp_config:
+                    if find_key in key and exp_config[key]["component_type_key"] == "SPLITTED_DATASET_ITERATORS":
+                        splits_percentage.append(exp_config[key]["config"]["split_configs"])
+                        for req in exp_config[key]["requirements"]:
+                            split_subscription.update(req["subscription"])
+                    elif exp_config[key]["component_type_key"] == "DATASET_ITERATORS":
+                        split_config = exp_config["dataset_iterators"]["config"]["split_configs"] if "split_configs" in exp_config["dataset_iterators"]["config"] else None
+
+                dataset_splits = {"split_config": split_config,
+                                  "split_subscription": list(split_subscription), 
+                                  "splits_percentage": splits_percentage}
```

Comment:
I think we discussed in our meeting two weeks ago, that we would leave this out for now no?

Reply:
Yes as I mentioned since the logic was already there I have kept it as optional information, If in the future you want to show it the option exists.

Reply:
I think this kind of information should come from MLgym. The same way we will be sending the number of prameters of a model from MLgym to the backend. Otherwise, the backend gets overloaded with MLgym internals